TutoringApp/App.js

```diff
@@ -53,6 +57,10 @@ export default function App() {
   );
 }
 
+function SignUpScreen(){ /** TESTING */
```

Comment:
If this is just for testing atm, before merging it to main I'd prefer if it were commented out, que no vaya a ser que en algun momento later on alguien haga un function llamado signup tambien and it gets called here by mistake.

Reply:
Clarification has been made. It calls for the render of the sign-up page itself. Therefore no need to comment it out since that will be a genuine necessity.